gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Driver/GremlinClientTests.cs

```diff
@@ -51,7 +52,18 @@ public async Task ShouldSendScriptForEvaluationAndReturnCorrectResult(string req
                 Assert.Equal(expectedResponse, response);
             }
         }
-
+        
+        [Fact]
+        public async Task ShouldReturnResponseAttributes()
+        {
+            var gremlinServer = new GremlinServer(TestHost, TestPort);
+            using (var gremlinClient = new GremlinClient(gremlinServer))
+            {
+                var requestMsg = _requestMessageProvider.GetDummyMessage();
+                var resultSet = await gremlinClient.SubmitAsync<object>(requestMsg);
+                Assert.NotNull(resultSet.StatusAttributes);
```

Comment:
@spmallette , with the tinkerpop server, I always get empty map here. I looked at the java tests for this, seems like you get "host" as one of the attributes. Am I missing something in the way I start the server ? I am using the start script in the bin of tinkerpop-server.

I did some manual testing with cosmos db server, it works fine.

Reply:
Sorry...the last rebase I did broke something. So, I fixed that, rebased again and added a few other minor changes. If you rebase your work on TINKERPOP-1913 I think things should start working here again.